<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mediaWindow</class>
 <widget class="QMainWindow" name="mediaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="songlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>19</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="lyricEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>351</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="searchPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="nextPushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>单曲播放</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>单曲循环</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>循环播放</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>随机播放</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="playProcessSlider">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="durationLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
